### DISCUSSION TITLE
Video will not import properly

### DISCUSSION THREAD

**Discussion by Clxverstxne** [keiwando/toonsquid#592] [General]
This issue is fairly recent where Toonsquid will have an issue importing my selected video into a project. Audio works just fine, however, the video will not show up and acts like it's not there.

The opacity is at 100% and I’ve tried to add other videos with the same issue (I’ve also imported the same video to Flipaclip, Youtube, and Capcut with no problems)
https://github.com/keiwando/toonsquid/assets/171306260/a87a9cca-775a-4bbc-9ec0-6f3ebc215429

**Comment by keiwando** [comment]
So this worked fine before but all of a sudden stopped working for all videos? How recently did this happen?
Also, what is the iPad model and the iOS version that you are using?

**Reply by Clxverstxne** [reply]
Yes, at least when I tried to import the video the other day (May 28th), before that I had imported about a 3-minute long video about a week before this.
iPad OS version 16.6 and it’s a 6th generation iPad

**Reply by keiwando** [reply]
Thanks for the info! 
Could you please send me a copy of that project file with the imported video? I can look into that to see if the video file gets broken during the import or if this is a playback issue. You can send the file to contact@toonsquid.com or upload it here.

**Reply by Clxverstxne** [reply]
https://drive.google.com/file/d/1haZKk4HZB1hyus1vhPRUn2yRa_L_Ov8q/view?usp=drivesdk Let me know if this works!

**Reply by keiwando** [reply]
Thank you! I can confirm that the video file in the project is not damaged in any way. I have imported your project onto several iPads (including one running iOS 16.6) but the video always shows up for me. 
Given this and the fact that you say that everything was working fine a couple of weeks ago and there was no new ToonSquid update since then, my current guess is that this might be an issue with the iPad's own media services, which is what ToonSquid uses to decode the video files.

Could you please try to manually [restart your iPad](https://support.apple.com/en-us/101603) and see if that happens to fix the issue? I've heard of rare instances where those media services end up in some broken state that can only be fixed by restarting the device.

**Reply by Clxverstxne** [reply]
Oh my, I was just shutting it off and turning it back on this entire time </3 this fixed the issue entirely, thank you so much!!!
